types_db.ts

```diff
@@ -7,6 +7,31 @@ export type Json =
   | Json[]
 
 export interface Database {
+  graphql_public: {
```

Comment:
To make this cleaner, you can remove `graphql_public` and `storage` from `api.schemas` in supabase/config.toml, or alternatively run the typegen command like `npx supabase gen types typescript [--local|--project-id ...]  --schema public > types_db.ts`

Reply:
Good catch, thanks!